.coveragerc

```diff
@@ -0,0 +1,12 @@
+[run]
```

Comment:
sorry, but could you put the coverage stuff in a separate PR?

Reply:
bump.

Reply:
(Isn't that https://github.com/matrix-org/synapse/pull/4180?)

Reply:
yes, #4180 :) (If we merge that first then this squashed merge won't have it, which is why I didn't bother removing it)